src/Altinn.Platform/Altinn.Platform.Common/Common/Common/Model/Actor.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Common.Model
+{
+    /// <summary>
+    /// Model for an Actor
+    /// </summary>
+    [Serializable]
+    public class Actor
+    {
+        /// <summary>
+        /// Gets or sets the PartyID
+        /// </summary>
+        [JsonProperty]
+        public int PartyID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the OrganizationNumber
+        /// </summary>
+        [JsonProperty]
+        public string OrganizationNumber { get; set; }
+
+        /// <summary>
+        /// Gets or sets the UnitType
+        /// </summary>
+        [JsonProperty]
+        public string UnitType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the SSNNumber
+        /// </summary>
+        [JsonProperty]
+        public string SSNNumber { get; set; }
```

Comment:
SSNNumber is repreating yourself.. Should be updated both in bridge code and here. SSN is used in the other bridge models